decidim-admin/app/controllers/decidim/admin/participatory_process_steps_controller.rb

```diff
@@ -68,14 +68,25 @@ def destroy
         redirect_to participatory_process_path(@participatory_process_step.participatory_process)
       end
 
+      def ordering
+        authorize ParticipatoryProcessStep
+
+        ReorderParticipatoryProcessSteps.call(collection, params[:items_ids]) do
+          on(:invalid) do
+            flash.now[:alert] = I18n.t("participatory_process_steps.ordering.error", scope: "decidim.admin")
+            redirect_to participatory_process_path(participatory_process)
```

Comment:
What you should do is put a spinner when you start the request, and delete that spinner when it finishes. But hey, `ujs` is a hack, this won't last long as we'll replace this with React+GraphQL (already working on it).